Index: src/sample/controllers/PersonOverviewController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/controllers/PersonOverviewController.java b/src/sample/controllers/PersonOverviewController.java
--- a/src/sample/controllers/PersonOverviewController.java	(revision 0987fc905e7c87bbcf6f49f2b43cbffa11a8893e)
+++ b/src/sample/controllers/PersonOverviewController.java	(date 1613723271425)
@@ -6,6 +6,7 @@
 import javafx.scene.control.TableView;
 import sample.Main;
 import sample.modules.Person;
+import sample.utils.DateUtil;
 
 public class PersonOverviewController {
     @FXML
@@ -44,7 +45,7 @@
         firstNameColumn.setCellValueFactory(cellData -> cellData.getValue().getFirstNameProperty());
         lastNameColumn.setCellValueFactory(cellData -> cellData.getValue().getLastNameProperty());
 
-        showPersonData(null);
+//        showPersonData(null);
 
         personTableView.getSelectionModel().selectedItemProperty().addListener(
                 (observable, oldValue, newValue) -> showPersonData(newValue)
@@ -54,20 +55,19 @@
     private void showPersonData(Person person) {
         if (person != null) {
             firstNameLabel.setText(person.getFirstName());
-            lastNameColumn.setText(person.getLastName());
+            lastNameLabel.setText(person.getLastName());
             streetLabel.setText(person.getStreet());
             postalCodeLabel.setText(Integer.toString(person.getPostalCode()));
             cityLabel.setText(person.getCity());
 
-            // birthdayLabel.setText(person.getBirthday());
+            birthdayLabel.setText(DateUtil.format(person.getBirthday()));
         } else {
             firstNameLabel.setText("");
-            lastNameColumn.setText("");
+            lastNameLabel.setText("");
             streetLabel.setText("");
             postalCodeLabel.setText("");
             cityLabel.setText("");
+            birthdayLabel.setText("");
         }
     }
-
-//    public void setMainapp
 }
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 0987fc905e7c87bbcf6f49f2b43cbffa11a8893e)
+++ b/.gitignore	(date 1613756064230)
@@ -1,8 +1,5 @@
 # Project exclude paths
 /out/
-.idea
-*.iml
-JavaFX-learning.iml
 ### Java template
 # Compiled class file
 *.class
Index: src/sample/utils/DateUtil.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sample/utils/DateUtil.java b/src/sample/utils/DateUtil.java
--- a/src/sample/utils/DateUtil.java	(revision 0987fc905e7c87bbcf6f49f2b43cbffa11a8893e)
+++ b/src/sample/utils/DateUtil.java	(date 1613665792284)
@@ -26,7 +26,7 @@
         }
     }
 
-    public static boolean validate(String dateString){
+    public static boolean validDate(String dateString){
         return DateUtil.parse(dateString) != null;
     }
 }
